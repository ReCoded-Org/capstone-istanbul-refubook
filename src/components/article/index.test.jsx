import React from 'react';
import Article from './';
import renderer from 'react-test-renderer';
it('Blog page renders correctly', () => {
  const tree = renderer.create(<Article />).toJSON();
  expect(tree).toMatchSnapshot();
});
