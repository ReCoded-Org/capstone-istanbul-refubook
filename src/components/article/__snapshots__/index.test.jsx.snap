// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog page renders correctly 1`] = `
<div
  className=" "
>
  <article
    className=""
  >
    <header>
      <h1
        className="text-center p-4 text-xl font-bold"
      >
        article.title
      </h1>
    </header>
    <section
      className="content-center"
    >
      <img
        alt="Article Cover"
        className="mx-auto"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "ImageHolder.svg",
          }
        }
      />
      <div
        className="flex items-center p-2 justify-center"
      >
        <img
          alt="Avatar of Jonathan Reinink"
          className="w-8 h-8 rounded-full mr-2"
          src={
            Object {
              "ReactComponent": Object {
                "$$typeof": Symbol(react.forward_ref),
                "render": [Function],
              },
              "__esModule": true,
              "default": "AuthorAvatar.svg",
            }
          }
        />
        <div
          className="text-xs"
        >
          <p
            className="text-gray-900 leading-none"
          >
            <a
              href="#author"
              target="_blank"
            >
              article.author.name
            </a>
          </p>
          <p
            className="text-gray-600"
          >
            Aug 18
          </p>
        </div>
      </div>
      <h2
        className="text-center pt-3 text-lg font-medium"
      >
        article.subTitle
      </h2>
    </section>
    <section
      className=""
    >
      <article
        className=" text-center"
      >
        <p
          className="p-3"
        >
          article.body
        </p>
        <p
          className="p-3"
        >
          article.body
        </p>
        <p
          className="p-3"
        >
          article.body
        </p>
        <footer
          className="text-center text-xs text-gray-600"
        >
          <p>
            <span>
               
              article.PostedOn
               
            </span>
            <time
              dateTime="article.dateTime"
            >
              Aug 18
            </time>
             
            article.by
             
            <a
              className="text-blue-700"
              href="#author"
            >
              article.author.name
            </a>
          </p>
        </footer>
      </article>
    </section>
  </article>
</div>
`;
